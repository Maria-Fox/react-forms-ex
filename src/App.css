.App {
  text-align: center;
}

.boxForm{
  display: flex;
  flex-direction: column;
}