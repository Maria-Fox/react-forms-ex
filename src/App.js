import logo from './logo.svg';
import BoxList from './BoxList';
import ToDoList from './ToDoList';
import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Color Boxes</h1>
      <BoxList />
      <ToDoList />
    </div>
  );
}

export default App;
