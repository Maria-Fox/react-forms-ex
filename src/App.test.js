import { render, screen } from '@testing-library/react';
import App from './App';

it('renders without crashing', () => {
  render(<App />);
});

it("matches snapchot", function () {
  const { asFragment, getByText } = render(<App />);

  const header = getByText("Color Boxes");
  
  expect(asFragment()).toMatchSnapshot();
  expect(header).toBeInTheDocument();
})