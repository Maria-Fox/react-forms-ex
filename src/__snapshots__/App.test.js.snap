// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapchot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Color Boxes
    </h1>
    <div>
      <form
        class="boxForm"
      >
        <p>
          Create a Box
        </p>
        <label
          for="height"
        >
           Box Height
          <input
            id="height"
            name="height"
            placeholder="89"
            required=""
            type="number"
            value=""
          />
        </label>
        <label
          for="width"
        >
           Box Width
          <input
            id="width"
            name="width"
            placeholder="14"
            required=""
            type="number"
            value=""
          />
        </label>
        <label
          for="backgroundColor"
        >
           Box Background Color
          <input
            id="backgroundColor"
            name="backgroundColor"
            placeholder="peachpuff"
            required=""
            type="text"
            value=""
          />
        </label>
        <button
          class="NewBoxForm-button"
        >
          Create Box!
        </button>
      </form>
      <div
        style="height: 88px; width: 90px; background-color: peachpuff;"
      >
        <button>
           X 
        </button>
      </div>
      <div
        style="height: 78px; width: 77px; background-color: pink;"
      >
        <button>
           X 
        </button>
      </div>
    </div>
    <div
      style="background-color: peachpuff;"
    >
      <p>
        Todo List
      </p>
      <form>
        <label
          for="todo"
        >
           Add a new todo item
          <input
            id="todo"
            name="todo"
            placeholder="wash dishes"
            type="text"
            value=""
          />
        </label>
        <button
          type="submit"
        >
          Submit
        </button>
      </form>
      <ul>
        <li>
          clean the house
          <button>
            X
          </button>
        </li>
        <li>
          study
          <button>
            X
          </button>
        </li>
        <li>
          work on react stuff
          <button>
            X
          </button>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
